src/bunyanStream.js

```diff
@@ -0,0 +1,82 @@
+const _ = require('lodash');
+const {Writable} = require('stream');
```

Comment:
```suggestion
const { Writable } = require('stream');
```
Linter should be complaining about this, no? Seems to be inconsistent.

Reply:
Once I started using r7, it complains if there is spaces around var

Reply:
not a point of preference. It's a point of what the linter allows...